content/en/developers/guide/unified-tagging-advanced-usage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Unified Tagging Advanced Usage Guide
+kind: guide
+aliases:
+  - /developers/faq/unified-tagging-advanced-usage
+---
+
+## Overview
+
+This guide suggests ways to configure and migrate to [unified service tagging][1] based on specific use cases.
+
+## Using Standard Labels
+
+The standard labels can be seen as an alternative to using environment variables, particularly for applications that will not benefit from using those variables in their runtime. 3rd party software like Redis, MySQL, and Nginx are a few examples. Since these services still generate infrastructure metrics and data from integration checks, it's valuable to have all of that data tagged with `env`, `service`, and `version`.
+
+If you do not use Docker as your container runtime on Kubernetes then you will have to use the standard labels to tag Kubernetes metrics. The Datadog Agent does not yet support the extraction of environment variables as tags from container runtimes outside of Docker.
+
+Lastly, if you would like to tag Kubernetes state metrics with `env`, `service`, and `version` then the standard labels offer the easiest way. The DD environment variables of a container cannot be used by the Agent to tag these metrics, so the labels are a more natural option.
```

Comment:
The agent can map env vars to tags if Docker is the container runtime. This is why we can recommend using just env vars for Docker generally.

We endorse standard labels in Kubernetes since users might have container runtimes other than Docker (e.g., `containerd`).